geniza/corpus/dates.py

```diff
@@ -126,6 +126,27 @@ def numeric_format(self, mode="min"):
         See :meth:`isoformat` for more details."""
         return self.isoformat(mode, "numeric")
 
+    @staticmethod
+    def get_date_range(old_range, new_range):
+        """Compute the min and max dates between two PartialDate ranges."""
```

Comment:
this is the widest possible range covered by the two ranges, correct? if you think of the date ranges as sets, this is the union rather than the intersection?

asking to make sure I understand - I think a slight wording change in the comment would make this clear.